### DISCUSSION TITLE
Country of origin of the HSR

### DISCUSSION THREAD

**Discussion by akinobu1998** [HumanSupportRobot/QandA#9] [Q&A] [answered]
Is the country of origin of the HSR Japan?
While we were preparing the international shipment of the HSR, the transportation company asked about this. I think it is Japan (i.e. designed in Japan and assembled in Japan). This is just a confirmation.

**Comment by hsr-admin** [comment, marked as answer]
As you may have guessed, HSRB is made in Japan, designed in Japan, and assembled in Japan.
